.github/workflows/remote-integ-tests-workflow.yml

```diff
@@ -132,6 +128,6 @@ jobs:
         uses: cypress-io/github-action@v2
         with:
           working-directory: opensearch-dashboards-functional-test
-          command: yarn run cypress run --env SECURITY_ENABLED=false --spec cypress/integration/plugins/anomaly-detection-dashboards-plugin/*.js
+          command: yarn run cypress run --env SECURITY_ENABLED=false --spec cypress/integration/plugins/anomaly-detection-dashboards-plugin/**/*.js
```

Comment:
Will this skip existing specs like [create_detector_spec.js](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/integration/plugins/anomaly-detection-dashboards-plugin/create_detector_spec.js) as you only execute specs under anomaly-detection-dashboards-plugin's subdirectories?

Reply:
No, they will be included along with any subdirectories - you can see that in the list of found specs in [comment above](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/560#issuecomment-1641146136)

Reply:
Your command in the comment has no .js: 

yarn cypress:run-without-security --spec 'cypress/integration/plugins/anomaly-detection-dashboards-plugin/**/*

Reply:
Confirmed it still shows 11 w/ command:
```
  │ Specs:          11 found (plugins/anomaly-detection-dashboards-plugin/create_detector_spec.js, │
  │                  plugins/anomaly-detection-dashboards-plugin/dashboard_spec.js, plugins/anomal │
  │                 y-detection-dashboards-plugin/detector_configuration_spec.js, plugins/anomaly- │
  │                 detection-dashboards-pl...)   
```